statusnet/statusnet.php

```diff
@@ -1426,7 +1427,12 @@ function statusnet_convertmsg(App $a, $body)
 
 			Logger::log("statusnet_convertmsg: expanding url " . $match[1], Logger::DEBUG);
 
-			$expanded_url = DI::httpClient()->finalUrl($match[1]);
+			try {
+				$expanded_url = DI::httpClient()->finalUrl($match[1]);
+			} catch (TransferException $exception) {
+				Logger::notice('statusnet_convertmsg: Couldn\'t get final URL.', ['url' => $match[2], 'exception' => $exception]);
+				$expanded_url = $match[2];
```

Comment:
Should still be `$match[1]`.
```suggestion
				$expanded_url = $match[1];
```